content/doc/book/blueocean/activity.adoc

```diff
@@ -18,32 +18,69 @@ repository Pipeline.
 
 image:blueocean/activity/overview.png[Overview of the Activity View, role=center]
 
-
 == Activity
 
+The default subsection of Activity Viewer shows a list of the latest
+completed or in-progress Runs of a Pipeline.
+Each line in the list corresponds to a Run of the Pipeline, showing
+the status of the run, id number, commit information, duration, and when the run completed.
+Clicking on a Run will bring up the <<pipeline-run-details#, Pipeline Run Details>>
+for that Run.
+"In Progress" Runs can be aborted from this list by clicking on the "Stop" symbol
+(a square "&#9724;" inside a circle).
+Runs that have completed can be re-run by clicking the
+"Re-run" symbol (a counter-clockwise arrow "&#8634;").
+The list can be filtered by branch or pull request by clicking on the "branch"
+drop-down in the list's header.
 
+This list does not allow runs to be edited or marked as favorites, those actions can be done from the
+"<<#branches, branches>>" subsection.
 
 == Branches
 
+The "Branches" subsection shows a list of all branches that have a
+completed or in-progress Run in the current Pipeline.
+Each line in the list corresponds to a branch in SCM, showing
```

Comment:
Should the first use of the acronym "SCM" use the expansion of the term followed by the acronym (as in, "software configuration management system (SCM))?